brave/brave-ios
# Fix #8663: Account Details v2

## Summary of Changes
- Updates Account Activity / Account Details to v2 designs.

This pull request fixes #8663

## Submitter Checklist:


## Test Plan:
1. Test opening account details for each coin type (Ethereum, Solana, Filecoin).
2. Verify total fiat balance loads in
3. Verify assets list loads with balances & prices for the selected account
4. Verify NFTs grid loads with metadata (if account has NFTs) for the selected account
5. Verify Transactions list loads in as expected for the selected account
6. Verify Dapp Settings opens for the coin type (row should be hidden for Filecoin accounts as they don't support DApps).
7. Tap `...` and tap `Edit`, verify edit account name modal is shown (this view will be replaced in the future; deposit will have it's own UI)
8. Update the account's name & verify name updates on Account Details.
9. Tap `...` and tap `Export`, verify export private key modal is shown.


## Screenshots:

Account Details | Account Details Dark
--|--
![Eth Account Details - light](https://github.com/brave/brave-ios/assets/5314553/3b66c445-9289-4c12-b9e3-120bcbad7bd3) | ![Eth Account Details - dark](https://github.com/brave/brave-ios/assets/5314553/2dfc14b8-fd00-4ca0-9a4c-c3618976f9a6)
Action Menu | Assets list
![Eth Account Details - action menu](https://github.com/brave/brave-ios/assets/5314553/f0aa90a9-8996-4541-957e-35f36deb523b) | ![Eth Account Details - assets - light](https://github.com/brave/brave-ios/assets/5314553/96164722-52c6-43c1-aa86-4cd14687b600)
NFT grid | Transactions List
![Eth Account Details - nts - light](https://github.com/brave/brave-ios/assets/5314553/39e40cc5-45a1-47a1-9a7b-dee818058805) | ![Eth Account Details - transactions - light](https://github.com/brave/brave-ios/assets/5314553/01b9d508-5d60-419c-9deb-49079a20217d)
DApps Connections | Filecoin (no dapp support)
![Eth Account Details - dapp connections - light](https://github.com/brave/brave-ios/assets/5314553/63150926-639d-4755-85b2-2eeae5ea4dc5) | ![Fil Account Details](https://github.com/brave/brave-ios/assets/5314553/c242f5d1-f01c-4045-8c0a-953987c3ffed)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
